team/bundles/org.eclipse.team.core/src/org/eclipse/team/internal/core/UserStringMappings.java

```diff
@@ -18,56 +19,64 @@
 import java.util.Iterator;
 import java.util.Map;
 import java.util.NoSuchElementException;
+import java.util.Optional;
 import java.util.StringTokenizer;
+import java.util.stream.Stream;
 
 import org.eclipse.core.runtime.Assert;
-import org.eclipse.core.runtime.Preferences;
-import org.eclipse.core.runtime.Preferences.PropertyChangeEvent;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.preferences.IEclipsePreferences;
+import org.eclipse.core.runtime.preferences.IEclipsePreferences.PreferenceChangeEvent;
+import org.eclipse.core.runtime.preferences.InstanceScope;
 import org.eclipse.team.core.Team;
+import org.osgi.service.prefs.BackingStoreException;
 
+public class UserStringMappings {
```

Comment:
in eclipse you need add Test classes to a suite - so that it the test is executed.

Reply:
Hello! Changed test class implementation to JUnit 3 and added its suite to AllTeamTests. Is there something left to be done regarding new test inclusion?

Reply:
good. it works:
https://ci.eclipse.org/platform/job/eclipse.platform/job/PR-549/5/testReport/org.eclipse.team.tests.core/UserMappingTest/